appengine/src/main/java/com/google/googleinterns/gscribe/modules/ServicesModule.java

```diff
@@ -43,7 +49,7 @@ public ExamService examParserServiceProvider(SpreadsheetService spreadsheetServi
     @Provides
     @Singleton
     public TokenService tokenServiceProvider(GoogleClientSecrets googleClientSecrets, NetHttpTransport netHttpTransport) {
-        return new TokenServiceImpl(googleClientSecrets, netHttpTransport);
+        return new TokenServiceImpl(googleClientSecrets, netHttpTransport, actionsClientID);
```

Comment:
Optional: You could have added an @Inject constructor, and have TokenServiceImpl as an argument, then directly return that.